cli/main.rs

```diff
@@ -285,8 +287,13 @@ pub fn main() {
       }
       Err(err) => unwrap_or_exit(Err(AnyError::from(err))),
     };
-    if !flags.v8_flags.is_empty() {
-      init_v8_flags(&flags.v8_flags);
+
+    let env_v8_flags = env::var("DENO_V8_FLAGS")
+      .ok()
+      .map(|flags| flags.split(',').map(String::from).collect::<Vec<String>>())
+      .unwrap_or_default();
+    if !flags.v8_flags.is_empty() || !env_v8_flags.is_empty() {
```

Comment:
This condition could maybe be moved into init_v8_flags so that it's shared by both standalone.rs and main.rs

Reply:
`init_v8_flags` is only used in main.rs, not by standalone.rs

Reply:
I meant this in the context of https://github.com/denoland/deno/pull/17313#discussion_r1065173839

Reply:
done